common.blocks/clearfix/clearfix.ru.md

```diff
@@ -0,0 +1,69 @@
+# clearfix
+
+Блок реализует популярный CSS-прием **clearfix**, также известный как [Easy Clearing Hack](http://www.456bereastreet.com/archive/200603/new_clearing_method_needed_for_ie7/). Прием позволяет отменить обтекание для элементов с CSS-свойством `float`, без внесения изменений в исходную HTML-структуру документа.
+
+Блок реализован в технологии CSS и подходит для использования в клиентских приложениях.
+
+Блок можно использовать в качестве контейнера для элементов со свойством `float`, или примешивая его к такому контейнеру.
+
+Пример использования в качестве контейнера:
+
+```bemjson
+({
+    block: 'header',
+    attrs: { style: 'border: 2px blue' },
```

Comment:
such `border` is invalid, border-type should be specified as well: `border: 2px solid blue` (same for other examples)